examples/firmware_drivers/main.tf

```diff
@@ -0,0 +1,16 @@
+provider "oneview" {
+	ov_username   = var.username
+	ov_password   = var.password
+	ov_endpoint   = var.endpoint
+	ov_sslverify  = var.ssl_enabled
+	ov_apiversion = 3000
+	ov_ifmatch    = "*"
+  }
+  
+# Creating Custom Firmware Service Pack
+resource "oneview_firmware_drivers" "drivers" {
+  baseline_uri = "/rest/firmware-drivers/Synergy_Custom_SPP_2021_02_01_Z7550-97110"
```

Comment:
can you use a GET call from data source and use it here? 
Instead of hard coding uri, use data source file to fetch the uri, will help in automation.